lib/shared/permissions.js

```diff
@@ -198,6 +198,13 @@ exports.elevateCommand = (command, options) => {
   // for now, we should make sure we double check if the error messages
   // have changed every time we upgrade `sudo-prompt`.
 
+  }).catch((error) => {
+    return _.includes(_.trim(error.message), 'is not in the sudoers file');
```

Comment:
Looks like the error message contains some ugly new lines I wanted to protect ourselves from.